.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule ".submodules/okta-sdk-test-server"]
```

Comment:
Pulls in IT mock dependencies.  When this project is more stable, we can pull them in with webjars. Until then, this config, and the corresponding pom.xml, make using git submodules transparent (or as much as that is possible)